src/api/src/main.py

```diff
@@ -26,11 +33,14 @@ def home():
 
 @app.get("/download/{story_id}")
 async def download_book(
-    story_id: int,
+    story_id: str,
     download_images: bool = False,
     username: Optional[str] = None,
     password: Optional[str] = None,
 ):
+
+    story_id=extract_story_ID(story_id)
```

Comment:
The logic is simple enough to not warrant placing this code in a function.

I'm yet to write a contributions.md, but I'd appreciate if you used `black` to format your code!